tests/commands/test-kob-upgrade.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+function __test_kob_init
+{
+    if [[ ! -d $KOBMAN_DIR ]]; then
+        echo "kobman not found"
+        echo "Please install kobman first and try again"
```

Comment:
@asa1997 The test should fail fast, Need not be interactive can exit with fail if any of the conditions checks are not statisfied.